src/Traits/HasRoles.php

```diff
@@ -51,9 +51,9 @@ public function roles(): MorphToMany
     /**
      * Scope the model query to certain roles only.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param string|array|\Spatie\Permission\Contracts\Role|\Illuminate\Support\Collection $roles
-     * @param string $guard
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  string|array|\Spatie\Permission\Contracts\Role|\Illuminate\Support\Collection  $roles
+     * @param  string  $guard
```

Comment:
Why all the unnecessary double spaces in all the docblocks?

Reply:
These double spaces are part of PSR2 code style requirements. I reverted them back to how they were originally in the new commit.